PizzaOvenUI/Screen_AwaitStart.qml

```diff
@@ -36,6 +36,13 @@ Item {
         localStoneTemp = menuSettings.json.menuItems[foodIndex].stoneTemp;
     }
 
+    function update() {
+        rootWindow.displayedDomeTemp = menuSettings.json.menuItems[foodIndex].domeTemp;
+        rootWindow.displayedStoneTemp = menuSettings.json.menuItems[foodIndex].stoneTemp;
+        localDomeTemp = menuSettings.json.menuItems[foodIndex].domeTemp;
+        localStoneTemp = menuSettings.json.menuItems[foodIndex].stoneTemp;
+    }
```

Comment:
This new function allows you to send new cycles from the mobile app while the display is already showing the await start screen for another cycle. This was completely broken before and whatever temps/time was displayed would not change when a new cycle was loaded from the app.

Reply:
Awesome! Great find and fix!